### DISCUSSION TITLE
Workflows - E-Mail-Notofication to Article-Author

### DISCUSSION THREAD

**Discussion by HusiBT** [joomla/joomla-cms#47072] [2. Cold Features]
Maybe this is not an issues, but would be a big benefit/feature for Workflows.
At Workflow´s TRANSITION -> Notification/Messages I can only enter a usergroup or an individual user for getting notification. Also this user needs to have notification enabled. 

Good feature would be to select the author of the article to inform by e-mail (even withouT user-setting "notificatoon: yes"), when a transition is done.

### Expected result
Being able to enter information-type "AUTHOR OF ARTICLE", not only usergroup or an individual user

Author of an article is informed by e-mail, when a certain workflows-transition is done.

### System information (as much as possible)
4.1.4

**Comment by stimpsonjcat** [comment]
I second this!

**Comment by chmst** [comment]
Discussion - idea - https://github.com/joomla/joomla-cms/discussions/38718
